src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ILambdaExpression.cs

```diff
@@ -0,0 +1,116 @@
+﻿using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    internal partial class IOperationTests : SemanticModelTestBase
+    {
+        [Fact]
+        public void ILambdaExpression_BoundLambda_HasValidLambdaExpressionTree()
+        {
+            string source = @"
+using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        Action x /*<bind>*/= () => F()/*</bind>*/;
+    }
+
+    static void F()
+    {
+    }
+}
+";
+            string expectedOperationTree = @"
+IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'Action x /* ... *</bind>*/;')
+  IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'Action x /* ... *</bind>*/;')
+    Variables: Local_1: System.Action x
+    Initializer: IConversionExpression (ConversionKind.CSharp, Implicit) (OperationKind.ConversionExpression, Type: System.Action) (Syntax: '() => F()')
+        ILambdaExpression (Signature: lambda expression) (OperationKind.LambdaExpression, Type: null) (Syntax: '() => F()')
+          IBlockStatement (2 statements) (OperationKind.BlockStatement) (Syntax: 'F()')
+            IExpressionStatement (OperationKind.ExpressionStatement) (Syntax: 'F()')
+              IInvocationExpression (static void Program.F()) (OperationKind.InvocationExpression, Type: System.Void) (Syntax: 'F()')
+            IReturnStatement (OperationKind.ReturnStatement) (Syntax: 'F()')
+";
+            var expectedDiagnostics = DiagnosticDescription.None;
+
+            VerifyOperationTreeAndDiagnosticsForTest<EqualsValueClauseSyntax>(source, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [Fact]
+        public void ILambdaExpression_UnboundLambdaAsVar_HasValidLambdaExpressionTree()
+        {
+            string source = @"
+using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        var x /*<bind>*/= () => F()/*</bind>*/;
+    }
+
+    static void F()
+    {
+    }
+}
+";
+            string expectedOperationTree = @"
+IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement, IsInvalid) (Syntax: 'var x /*<bi ... *</bind>*/;')
+  IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'var x /*<bi ... *</bind>*/;')
+    Variables: Local_1: var x
+    Initializer: ILambdaExpression (Signature: lambda expression) (OperationKind.LambdaExpression, Type: null) (Syntax: '() => F()')
```

Comment:
I think it is fine to have no conversion in ```var``` case in C#. In VB, it is different, the code is not an error and there is a n inferred delegate type, I also doubt there is an UnboundLambda node in VB bound tree in this scenario.